mappings/net/minecraft/container/ContainerWorldContext.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS aud net/minecraft/container/ContainerWorldContext
+	CLASS aud$1
+		METHOD a (Ljava/util/function/BiFunction;)Ljava/util/Optional;
+			ARG 1 arg1
+	CLASS aud$2
+		METHOD a (Ljava/util/function/BiFunction;)Ljava/util/Optional;
+			ARG 1 arg1
+	FIELD a noOpContext Laud;
```

Comment:
Not static final?

Reply:
yeah, it's a static final field, did enigma save it wrong?

Reply:
No, I mean, for constants we use NO_OP_CONTEXT or something